Readme.md

```diff
@@ -1,87 +1,106 @@
 # SQLite
 
-Sqlite fork ported to WebAssembly and adapted for the Fluence network. Could be launched and played on the Fluence [dashboard](http://dash.fluence.network/deploy/sqlite).
 
-Based on SQlite version 3.40.1.
+## Overview
+SQLite fork adapted to work with Fluence protocol. The current version is based on SQLite version 3.40.1. The artifact is SQLite WASM module:
+
+
+- that is compiled following [Marine runtime ABI conventions](https://fluence.dev/docs/marine-book/marine-rust-sdk/module-abi)
+- to be used with [Marine runtime](https://github.com/fluencelabs/marine)
+
+
+## How to build
+
+
+
+Wasm module can be built with either docker-compose
 
-# How to build
 
-This app could be built either with docker
 
 ```bash
+
 docker-compose up
+
 ```
 
-or by Makefile with [wasi-sdk](https://github.com/WebAssembly/wasi-sdk) installed
+
+
+or using this [Makefile](./Makefile) with GNU make. There are prerequisites to be installed following this path, namely [wasi-sdk](https://github.com/WebAssembly/wasi-sdk) and marine crate. Take a look at the Dockerfile for the details.
+
+
```

Comment:
Isn't this obvious after GNU make was mentioned?

Reply:
This was a suggestion by @boneyard93501 as I got it.